docs/controls/file-browser/v8-schema.md

```diff
@@ -24,6 +24,30 @@ This outlines the full schema available to configure the btowser. These options
          * The host app's authority, used as the target origin for post-messaging.
          */
         origin: string;
+        /**
```

Comment:
If we document these at all, we need to provide guidance on how to use them. Otherwise we should keep them "hidden".

Reply:
I can update the examples in a follow-up PR. I'll update my work items and get that done ASAP once this completes.